LocalMultiplayerAgent/MultiplayerServerManager.cs

```diff
@@ -163,6 +164,72 @@ private void ExtractAssets(string assetFileName, string targetFolder)
 
                 _systemOperations.ExtractToDirectory(assetFileName, targetFolder);
             }
+            else
+            {
+                _systemOperations.CreateDirectory(targetFolder);
+
+                ProcessStartInfo processStartInfo = Path.GetExtension(assetFileName).ToLowerInvariant() == Globals.ZipExtension
+                   ? GetProcessStartInfoForZip(assetFileName, targetFolder)
+                   : GetProcessStartInfoForTarOrGZip(assetFileName, targetFolder);
+
+                _logger.LogInformation($"Starting asset extraction with command arguments: {processStartInfo.Arguments}");
```

Comment:
>LogInformation [](http://example.com/codeflow?start=24&length=14)

Should this be Log Debug or Verbose?

Reply:
Thanks Pedro! I didn't put a lot of attention to Logging Levels and just learned there are different logging levels for different purposes. I think I can replace this to LogVerbose.

Reply:
For local multiplayer agent, I don't think it makes much difference. You can keep it as is. For the actual agent, info means it will be sent to Geneva (which should be fine for asset extraction since it only happens once on a VM). Verbose are only written to local files on the VM and never sent to GEneva.